eg/eg_exec.py

```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/python
 import argparse
-import eg_config
-import eg_util
 import pydoc
 import sys
 
+try:
+    import eg_config
+    import eg_util
+except ImportError:
+    from eg import eg_config
+    from eg import eg_util
+
```

Comment:
It looks like the problem stems from the fact that `eg_exec.py` is meant to be run as a script but lives in the `eg` package itself, which [according to Guido](http://stackoverflow.com/questions/16981921/relative-imports-in-python-3) is an antipattern.

Going to merge in the next day or two once I figure out if I agree.